velox/connectors/hive/HiveConfig.h

```diff
@@ -44,9 +44,6 @@ class HiveConfig {
 
   /// Maximum number of (bucketed) partitions per a single table writer
   /// instance.
-  ///
-  /// TODO: remove hive_orc_use_column_names since it doesn't exist in presto,
-  /// right now this is only used for testing.
   static constexpr const char* kMaxPartitionsPerWriters =
```

Comment:
@majetideepak How about moving the following file format-related configurations in HiveConfig into the file format's configuration file as well?

https://github.com/facebookincubator/velox/blob/main/velox/connectors/hive/HiveConfig.h#L76-L84

Reply:
@JkSelf I prefer to handle the ReaderOptions in a separate PR. The ReaderOptions are not specialized per file format similar to the WriterOptions